src/components/FlowRunMenu.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <p-icon-button-menu>
     <template #default>
+      <!-- todo: this PR(https://github.com/PrefectHQ/prefect-ui-library/pull/1743) will only partially work once object acls are on -->
```

Comment:
oops looks like this comment bled into my PR before I got around to addressing it. 

for context, I've marked several places like here where `can.*.deployment` checks are happening but no deployment object is available in-context. Some places are probably _fine_ - for example, `can.view.deployment` checks since we decided that `view` is a pre-requisite to gaining additional permissions via an ACL. Other places need some thought like this one. 

I'm thinking of having a follow-up PR with those places slash going through them all separately after this PR passes the "yeah this approach all checks out"